docusaurus/docs/user-docs/content-type-builder/creating-new-content-type.md

```diff
@@ -50,7 +56,7 @@ To create a new component:
 2. In the component creation window, configure the base settings of the new component:
    - Write the name of the component in the *Name* textbox.
    - Select an available category, or enter in the textbox a new category name to create one.
-   - Choose an icon to represent the new component.
+   - Choose an icon to represent the new component. You can use the search ![Search icon](/img/assets/icons/search.svg) to find an icon instead of scrolling through the list.
```

Comment:
```suggestion
   - (_optional_) Choose an icon representing the new component. You can use the search ![Search icon](/img/assets/icons/search.svg) to find an icon instead of scrolling through the list.
```